package/com.unity.formats.usd/Tests/USD.NET.Unity/UsdNetUnityTests.cs

```diff
@@ -0,0 +1,159 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using NUnit.Framework;
+using Unity.Formats.USD;
+
+namespace USD.NET.Unity.Tests
+{
+    class UsdTests
+    {
+        readonly List<string> m_filesToDelete = new List<string>();
+
+        [SetUp]
+        public void SetUp()
+        {
+            InitUsd.Initialize();
+        }
+
+        protected string CreateTmpUsdFile(string fileName)
+        {
+            var filePath = Path.Combine(UnityEngine.Application.dataPath, fileName);
+            var scene = Scene.Create(filePath);
+            scene.Save();
+            scene.Close();
+            m_filesToDelete.Add(filePath);
+            return filePath;
+        }
+
+        [TearDown]
+        public void DeleteTmpFiles()
+        {
+            foreach (var file in m_filesToDelete)
+            {
+                File.Delete(file);
```

Comment:
You Might want to delete the meta file as well if it exists. Otherwise you'll get a one time warning telling you it found a meta without the file

Reply:
Also you might delete the file from the "delete list" once it's deleted cause this function could be called multiple times in theory.